packages/ember-views/lib/views/view.js

```diff
@@ -1216,25 +1216,35 @@ var View = CoreView.extend({
   _applyAttributeBindings: function(buffer, attributeBindings) {
     var attributeValue;
     var unspecifiedAttributeBindings = this._unspecifiedAttributeBindings = this._unspecifiedAttributeBindings || {};
+    var attributeHash = {};
 
-    forEach(attributeBindings, function(binding) {
+    for (var i = attributeBindings.length-1; i >= 0; i--) {
+      var binding = attributeBindings[i];
       var split = binding.split(':');
       var property = split[0];
       var attributeName = split[1] || property;
 
       Ember.assert('You cannot use class as an attributeBinding, use classNameBindings instead.', attributeName !== 'class');
 
-      if (property in this) {
-        this._setupAttributeBindingObservation(property, attributeName);
+      if (!attributeHash.hasOwnProperty(attributeName)) {
```

Comment:
Should I only test this if `Ember.FEATURES.isEnabled("feature-name")` is true?
If so, which feature name should I use?